src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs

```diff
@@ -226,11 +240,24 @@ private void LoadItems(PackageItemListViewModel selectedPackageItem, Cancellatio
                     var errorMessage = ExceptionUtilities.DisplayMessage(ex);
                     _logger.Log(ProjectManagement.MessageLevel.Error, errorMessage);
 
-                    _loadingStatusIndicator.SetError(errorMessage);
-
                     _loadingStatusBar.SetError();
                     _loadingStatusBar.Visibility = Visibility.Visible;
                 }
+                finally
+                {
+                    if (_loadingStatusIndicator.Status != LoadingStatus.NoItemsFound)
```

Comment:
i thought when we talked on friday...we wanted to ensure that we don't remove the statusindicator if either of those 2 exceptions happen. your current check checks for "empty" list...does it handle those exceptions well. what about for multi-feed searches?

Reply:
I will revert the error messages in the exceptions.

Reply:
Multi-feed search can fail and lead to an error. Documented in this issue:   https://github.com/NuGet/Home/issues/9218